src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+html,
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__link {
+  height: 40px;
+  margin: auto 0;
+}
+
+.menu {
+  display: flex;
+  justify-content: end;
+  list-style: none;
+}
+
+.menu__item:not(:last-child) {
+  position: relative;
+  display: flex;
+  margin-right: 20px;
+}
+
+.menu__link.is-active {
+  color: #00acdc;
+}
+
+.menu__link.is-active::after {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  content: "";
+  width: 100%;
```

Comment:
If you set width: 100% - it means that it will take all width, and no need to set `left: 0`